testsuite/features/step_definitions/common_steps.rb

```diff
@@ -593,15 +593,22 @@
 When(/^I enable repositories before installing Docker$/) do
   os_version, os_family = get_os_version($minion)
 
-  # Distribution Pool and Update
+  # Distribution
   repos = "os_pool_repo os_update_repo"
   puts $minion.run("zypper mr --enable #{repos}")
 
   # Tools
   repos, _code = $minion.run('zypper lr | grep "tools" | cut -d"|" -f2')
   puts $minion.run("zypper mr --enable #{repos.gsub(/\s/, ' ')}")
 
-  # Container repositories
+  # Development
+  # (no Python 2 needed starting with version 4.1)
```

Comment:
I don't understand the meaning of that comment on the context of the lines bellow. Either we can just rid of that comment, or we need to reason it a bit more.

Reply:
It's because the code here is different from the 4.0 and 3.2 code. This is to explain why we intentionally don't install python 2 starting with version 4.1.

Reply:
Hopefully clarified

Reply:
If you want to have a look in branch 4.0, feel free.

I did not want to bother the reviewers with two slightly different branches...